src/swarmus_ros_simulation/launch/teleop_from_cell_multi_robot.launch

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+<!-- This launch files spawns a swarm of 5 fully functional robots -->
+
+<launch>
+    <include file="$(find swarmus_ros_simulation)/launch/reusable/simulation_services.launch"/>
+    <!-- Agent 1 -->
+    <include file="$(find swarmus_ros_simulation)/launch/reusable/simulated_agent.launch">
+        <arg name="agent_id"   value="1"/>
+    </include>
+    <include file="$(find swarmus_ros_simulation)/launch/reusable/pioneer_spawner.launch">
+        <arg name="ns" value="agent_1"/>
+        <arg name="robot_name" value="agent_1"/>
+    </include>
+
+    <!-- Agent 2 -->
+    <include file="$(find swarmus_ros_simulation)/launch/reusable/simulated_agent.launch">
+        <arg name="agent_id"   value="2"/>
+    </include>
+    <include file="$(find swarmus_ros_simulation)/launch/reusable/pioneer_spawner.launch">
+        <arg name="ns" value="agent_2"/>
+        <arg name="robot_name" value="agent_2"/>
+        <arg name="robot_pos_x" value="1.0"/>
+        <arg name="robot_pos_y" value="1.0"/>
+    </include>
+
+    <!-- Agent 3: Used by android cellphone -->
+    <include file="$(find swarmus_ros_simulation)/launch/reusable/simulated_cellphone_hiveboard_assembly.launch">
+        <arg name="agent_id"   value="3"/>
+        <!-- This address is the loopback address when running with emulator and the android device address otherwise -->
+        <arg name="hivemind_host_address" value="127.0.0.1" />
+        <!-- This port is either the one set on the device or the port that has been configured to forward traffic
+            to the emulator. See HiveAR readme.md to see how to do this -->
+        <arg name="hivemind_host_port" value="6901"/>
```

Comment:
isn't it 7001 instead of 6901? I don't recall seeing 6901 elsewhere

Reply:
We should really put all the ports in a document in confluence, so the default values in the launch files and the default values in the c++ code matches everywhere

Reply:
That was my port that forwarded traffic to my emulator, i'll change it to the default 7003 port for agent 3